### PR TITLE
Add member webhook

### PR DESCRIPTION
**Pull request by mgross-ebner**
Hi,

this PR adds group member events as documented [here](https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html#group-member-events).

Best regards
Markus

### CODE REVIEW

**Comment by mgross-ebner** [comment]
@svanharmelen
If there is anything missing except your time for a review, please let me know.

File: event_parsing.go [hunk 1]
```diff
@@ -195,20 +194,19 @@ func WebhookEventType(r *http.Request) EventType {
...
+//	    event, err := gitlab.ParseWebhook(gitlab.HookEventType(r), payload)
+//	    if err != nil { ... }
+//	    switch event := event.(type) {
+//	    case *gitlab.PushEvent:
+//	        processPushEvent(event)
+//	    case *gitlab.MergeEvent:
+//	        processMergeEvent(event)
+//	    ...
+//	    }
+//	}
```
[the remark this answers is not in the document]
Comment (mgross-ebner) [reply event_parsing.go:209]: I saw the change but assumed it was an adjustment by go-fmt. The comment itself should be unchanged.

[the remark this answers is not in the document]
Comment (svanharmelen) [reply event_parsing.go:209]: OK, then can you please revert that one? Rest looks good now 👍🏻

[the remark this answers is not in the document]
Comment (svanharmelen) [reply event_parsing.go:209]: Ow wait, it's formatted by `go-fmt` now? Then I'll revert it also fix the tests 👍🏻

**Pull request merged**